pkg/ebpf/c/common/kconfig.h

```diff
@@ -1,9 +1,11 @@
-#ifndef __TRACEE_KCONFIG_H__
-#define __TRACEE_KCONFIG_H__
+#ifndef __COMMON_KCONFIG_H__
+#define __COMMON_KCONFIG_H__
 
 #include <vmlinux.h>
+#include <vmlinux_flavors.h>
+#include <vmlinux_missing.h>
```

Comment:
I don't see a reason to include those headers here

Reply:
ditto (tried to create a pattern, can remove it).